databases/cockroachdb/prisma/schema.prisma

```diff
@@ -1,9 +1,45 @@
 generator client {
   provider        = "prisma-client-js"
-  previewFeatures = ["cockroachdb"]
 }
 
 datasource db {
   provider = "cockroachdb"
   url      = env("DATABASE_URL")
 }
+
+model User {
+  id        String    @id @default(uuid())
```

Comment:
Thanks @aydrian . I'm going to leave it as is for this PR since its primarily just updating our fork. I'll look at integrating the feedback into the next PR I send out.

Reply:
I see that these were made by folks outside of cockroach and seem applicable to the upstream prisma-examples repo. https://github.com/prisma/prisma-examples/pull/3691. I recommend we PR these back to the prisma-examples repo directly if they're important.